lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -27,10 +28,31 @@
 from lte.protos.mobilityd_pb2 import IPAddress
 from magma.pipelined.policy_converters import get_ue_ip_match_args, get_eth_type
 from magma.pipelined.openflow.registers import Direction
+from lte.protos.pipelined_pb2 import (
```

Comment:
To resolve the isort lint errors below: (in your dev VM)
```
sudo apt-get install -y isort
isort -y ~/magma/lte/gateway/python/magma/pipelined/app/classifier.py
```

Reply:
For classifier it looks fine...but the report sometimes showing error
-------------------------------------------------------------------------------
vagrant@magma-dev:~/magma$ isort -y ~/magma/lte/gateway/python/magma/pipelined/app/classifier.py
vagrant@magma-dev:~/magma$ git status
On branch topic/yogesh/gtp_pipeline_4554
vagrant@magma-dev:~/magma$
------------------------------------------------------------------------

Reply:
Report showing some open items...But while running locally it seems no issue found.